src/bentoml/models.py

```diff
@@ -257,14 +301,19 @@ def create(
         context=context,
     )
     external_modules = [] if external_modules is None else external_modules
-    imported_modules = []
+    imported_modules: t.List[ModuleType] = []
     try:
+        res.flush()
         res.enter_cloudpickle_context(external_modules, imported_modules)
         yield res
     except Exception as e:
         raise e
```

Comment:
I'm not sure why these are necessary either, isn't this identical to not having a try block?

Reply:
^